src/ddbToEs/ddbToEsHelper.ts

```diff
@@ -70,12 +72,27 @@ export default class DdbToEsHelper {
                                 },
                             },
                         },
+                        [`${indexName}-alias`]: {},
                     },
                 };
                 await this.ElasticSearch.indices.create(params);
+            } else {
```

Comment:
I think that the if conditions are slightly off.
If the index does not exists, then we should create the index AND the alias. (right now the alias is only created if the index existed before this method call)

something like:
```js
if (aliasExists){
	return
}

if(!indexExists){
	createIndex()
}

createAlias()
```

Reply:
The create index and alias scenario is handled by line 75, where the alias is created as part of the create index call.

Reply:
My bad.

Is line 75 correct? Shouldn't it be inside a "aliases" object?
i.e.
```js
"aliases": {
    [`${indexName}-alias`]: {},
  }
```

https://www.elastic.co/guide/en/elasticsearch/reference/7.x/indices-create-index.html#create-index-aliases

Reply:
Good catch!